src/Modal/consts.js

```diff
@@ -6,3 +6,10 @@ export const SIZES = {
 };
 
 export const CLOSE_BUTTON_DATA_TEST = "ModalCloseButton";
+
+export const FOCUSABLE_ELEMENT_SELECTORS =
+  'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, [tabindex="0"], [contenteditable]';
+
+export const KEY_CODE_MAP = {
```

Comment:
Could be `KEY_CODE_MAP` in some common file ? we should use it in also in another components